pep-0496-environment-markers.rst

```diff
@@ -73,8 +93,8 @@ Micro-language
 The micro-language behind this is as follows. It compares:
 
 * strings with the ``==`` and ``in`` operators (and their opposites)
-* version numbers with the ``<``, ``<=``, ``>=``, and ``<`` operators
-  in addition to those supported for strings
+* version numbers with the ``==``, ``!=``, ``<``, ``<=``, ``>=``, and ``<`` operators.
```

Comment:
Ho hum.

Haven't tested. Depends how those things are implemented. I suspect - since they're not standard python operators, but must (i assume) be specially handled in the code that parses dependencies - that they won't work. I'm not sure whether it's worth explicitly stating this: as far as I can recall they only have meaning when referring to python names, which we aren't doing here.